hangman/.gitignore

```diff
@@ -0,0 +1,20 @@
+# The directory Mix will write compiled artifacts to.
```

Comment:
Please don't remove lines from .gitignore. It's there to prevent git from adding files that shouldn't be saved in the repository (such as backups, binary .beam files and so on)